salt/templates/virt/libvirt_domain.jinja

```diff
@@ -71,31 +71,37 @@
                             {% endif %}/>
                 </graphics>
                 {% endif %}
-                {% if serial_type == 'pty' %}
+
+                {% for serial in serials %}
+                {% if serial.type == 'pty' %}
                 <serial type='pty'>
                         <target port='0'/>
                 </serial>
-                {% if console %}
+                {% endif %}
+                {% if serial.type == 'tcp' %}
```

Comment:
[Jinja](https://jinja.palletsprojects.com/en/2.10.x/templates/#if) has an `{% elif %}` block, please use it here.
```suggestion
                {% elif serial.type == 'tcp' %}
```